components/abstractions/src/main/java/com/microsoft/kiota/RequestInformation.java

```diff
@@ -152,29 +151,19 @@ public void setUri(@Nonnull final URI uri) {
      * Adds query parameters to the request based on the object passed in and its fields.
      * @param parameters The object to add the query parameters from.
      */
-    public void addQueryParameters(@Nullable final Object parameters) {
-        if (parameters == null) return;
-        final Field[] fields = parameters.getClass().getFields();
-        for (final Field field : fields) {
-            try {
-                Object value = field.get(parameters);
-                String name = field.getName();
-                if (field.isAnnotationPresent(QueryParameter.class)) {
-                    final String annotationName = field.getAnnotation(QueryParameter.class).name();
-                    if (annotationName != null && !annotationName.isEmpty()) {
-                        name = annotationName;
-                    }
-                }
+    public void addQueryParameters(@Nullable final QueryParameters parameters) {
```

Comment:
argh, forgot a line here:

```
if (parameters == null) return;
```

Reply:
arg, I missed it too... Please submit a patch (with patch bump for the version) as I just finished releasing it.

Reply:
noticed, and it's ready 😄